code/game/turfs/closed/wall/mineral_walls.dm

```diff
@@ -356,17 +356,20 @@
 /////////////////////Lavaland Base Syndicate Explosive Walls /////////////////////
 
 /turf/closed/wall/mineral/plastitanium/explosive
-	var/obj/item/bombcore/large/syndicate_base/bombcore
+	///Ensures the payload can only go off once, because of shenanigans where it didn't
+	var/payload_active = TRUE
 
-/turf/closed/wall/mineral/plastitanium/explosive/Initialize(mapload)
-	. = ..()
-	bombcore = new /obj/item/bombcore/large/syndicate_base(src)
+/turf/closed/wall/mineral/plastitanium/explosive/proc/try_detonate()
+	if(payload_active)
+		payload_active = FALSE
+		explosion(src, 12, 16, 20, 20, 20)
```

Comment:
Can you use named params for my sanity

Reply:
Yeah, np
I'm failing a boatload of checks for reasons I don't understand too.

Reply:
done